hoomd/data/typeparam.py

```diff
@@ -208,6 +208,7 @@ def to_base(self):
 
     def to_dict(self):
         """Alias for `to_base`."""
+        return self.to_base()
```

Comment:
Alternately, remove the whole `to_dict` method.

Reply:
@b-butler are the `to_dict` and `to_base` methods supposed to be user-facing?

Reply:
Yes, `to_base` is a user-facing method. It replaces the deprecated `to_dict`.

Reply:
Because the `hoomd.data.collection` classes have `to_base` method and not all of them are mapping types, so `to_base` allows a consistent API for converting to standard Python data types.